src/experimental/tools/mediaCapabilitiesProber/probers/mediaContentTypeWithFeatures/index.ts

```diff
@@ -81,6 +81,6 @@ export default function probeTypeWithFeatures(
       }
     }
 
-    return formatSupport(result);
+    return [formatSupport(result)] as [number];
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=315&issues=AWXy2D_H2IDoRcZ5Haqr&open=AWXy2D_H2IDoRcZ5Haqr)

[ISSUE_KEY]: AWXy2D_H2IDoRcZ5Haqr